--- /Users/Kevin/GitHub/RpiDisplaySetup.copy/FileSets/PatchSource/start-gui.sh.orig	2024-03-16 22:28:32
+++ /Users/Kevin/GitHub/RpiDisplaySetup.copy/FileSets/PatchSource/start-gui.sh	2024-03-23 07:52:33
@@ -65,6 +65,10 @@
 
 scriptdir=$(dirname "$BASH_SOURCE")
 
+##### modified for RpiDisplaySetup 
+##### reset brightness range for RPI display AFTER GUI starts 
+nohup sleep 1; dbus -y com.victronenergy.settings /Settings AddSettings '%[{"path": "/Gui/Brightness", "default":255, "min":0, "max":255}]' > /dev/null &
+
 if [ "$vnclocal" = "1" ] || [ "$vncinternet" = "1" ]; then
 	echo "*** Starting gui, with VNC enabled (VncLocal=$vnclocal VncInternet=$vncinternet)"
 	exec ${scriptdir}/gui $mouse -display "${multistring}VNC:size=$size:depth=32:passwordFile=/data/conf/vncpassword.txt:0"
